dashboard/app/entities_spanner.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+	"unicode"
+
+	"cloud.google.com/go/civil"
+	"cloud.google.com/go/spanner"
+	"google.golang.org/api/iterator"
+)
+
+// This file contains definitions of entities stored in spanner.
+
+type CoverageHistory struct {
+	instrumented map[string]int64
+	covered      map[string]int64
+}
+
+func nsFilesTable(ns string) string {
```

Comment:
By the way, what do you think of using just one table for all namespaces? Then namespace becomes a table field and you no longer need to worry about sanitizing its value.

Reply:
Looks reasonable.
The lowest granularity I see for the access management is a DB, not table. It was the main reason I wanted to use per-namespace tables.